### ISSUE TITLE
Setup Issue

### ISSUE THREAD

**Issue by cchandel-dev** [pyansys/pyaedt#1751]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

During the setup process, I ran the following command: `pip install -r PyAEDT/requirements_test.txt`
I got the error shown below
![MicrosoftTeams-image](https://user-images.githubusercontent.com/76410929/192831835-b9e5a47c-a7ae-43fd-999c-01fc9ab2616c.png)

This error was seen for numpy (as shown in the image above as well as scikit-learn and pandas). The error was resolved by changing the version that was being searched for in requirements_test.txt to match the last version searched for (shown in the error  message above as 1.21.6).

It looks like the newest versions of numpy, pandas and scikit-learn is no longer supported on python 3.7. 

### Steps To Reproduce

run the following command: `pip install -r PyAEDT/requirements_test.txt`

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.7

### Installed packages

aiohttp==3.8.3
aiosignal==1.2.0
anyio==3.6.1
appdirs==1.4.4
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
async-timeout==4.0.2
asynctest==0.13.0
atomicwrites==1.4.1
attrs==22.1.0
Babel==2.10.3
backcall==0.2.0
beautifulsoup4==4.11.1
bleach==5.0.1
certifi==2022.9.24
cffi==1.15.1
charset-normalizer==2.1.1
clr-loader==0.2.3
codecov==2.1.12
colorama==0.4.5
coverage==6.4.4
cycler==0.11.0
debugpy==1.6.3
decorator==5.1.1
defusedxml==0.7.1
entrypoints==0.4
et-xmlfile==1.1.0
fastjsonschema==2.16.2
fonttools==4.37.3
frozenlist==1.3.1
idna==3.4
imageio==2.22.0
importlib-metadata==4.12.0
importlib-resources==5.9.0
iniconfig==1.1.1
ipykernel==6.15.3
ipython==7.34.0
ipython-genutils==0.2.0
jedi==0.18.1
Jinja2==3.1.2
joblib==1.1.0
json5==0.9.10
jsonschema==4.16.0
jupyter-client==7.3.5
jupyter-core==4.11.1
jupyter-server==1.19.1
jupyterlab==3.4.7
jupyterlab-pygments==0.2.2
jupyterlab-server==2.15.2
kiwisolver==1.4.4
lxml==4.9.1
MarkupSafe==2.1.1
matplotlib==3.5.2
matplotlib-inline==0.1.6
mistune==2.0.4
multidict==6.0.2
nbclassic==0.4.3
nbclient==0.6.8
nbconvert==7.0.0
nbformat==5.6.1
nest-asyncio==1.5.5
notebook==6.4.12
notebook-shim==0.1.0
numpy==1.21.6
openpyxl==3.0.10
packaging==21.3
pandas==1.3.5
pandocfilters==1.5.0
parso==0.8.3
pickleshare==0.7.5
Pillow==9.2.0
pkgutil-resolve-name==1.3.10
pluggy==1.0.0
plumbum==1.7.2
prometheus-client==0.14.1
prompt-toolkit==3.0.31
psutil==5.9.2
py==1.11.0
-e git+https://github.com/myoung301/pyaedt/@2f167794901beda70a028610c66563467b9cd972#egg=pyaedt
pycparser==2.21
Pygments==2.13.0
pyparsing==3.0.9
pyrsistent==0.18.1
pytest==7.1.2
pytest-cov==3.0.0
python-dateutil==2.8.2
pythonnet==3.0.0rc4
pytz==2022.2.1
pyvista==0.36.1
pywin32==304
pywinpty==2.0.8
pyzmq==24.0.1
requests==2.28.1
rpyc==5.0.1
scikit-learn==1.0.2
scipy==1.7.3
scooby==0.6.0
Send2Trash==1.8.0
six==1.16.0
sklearn==0.0
sniffio==1.3.0
soupsieve==2.3.2.post1
terminado==0.15.0
threadpoolctl==3.1.0
tinycss2==1.1.1
tomli==2.0.1
tornado==6.2
traitlets==5.4.0
typing-extensions==4.3.0
urllib3==1.26.12
vtk==9.1.0
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.4.1
wslink==1.8.2
yarl==1.8.1
zipp==3.8.1

**Comment by Samuelopez-ansys** [comment]
Hi @cchandel-dev ,

Indeed, ofr CPython 3.7 you need pandas 1.3.5. The problem is your are installing the requirement_test.txt, which are the requirements pyaedt is using for the unittest (CI) (the virtual machine is testing with CPython 3.8 and Ironpython 2.7).

In your machine, you need to install the file requirements.txt:

pip install -r requirements.txt

If you open this file, you will see how it is correctly set:

![image](https://user-images.githubusercontent.com/85613111/193289408-bbe6aa18-8d64-499c-9110-060d83eb9034.png)

Please let me know if it is clear and close the issue.

Regards
